tests/test_a_consumos.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import requests
+import json
+
+url = 'http://127.0.0.1:5000/api/'
+endpointName = 'consumos'
+idTeste = 'tykEWk7L-A'
+infoTeste = {
+    "cod_cliente": "_bFWsv-uFg",
+    "cod_consumo": "tykEWk7L-A",
+    "cod_funcionario": "PYYAr9K67g",
+    "data_consumo": "Thu, 19 Dec 2019 15:49:54 GMT",
+    "n_ementas_consumidas": 1,
+    "total": "$9.00"
+}
+
+
+def test_case1_get():
+    response = requests.get(f"{url}{endpointName}/")
+    assert response.status_code == 200
+
+
+def test_case2_get():
```

Comment:
Test case failing.

![image](https://user-images.githubusercontent.com/39097691/71289421-f7d73780-2364-11ea-92bc-0583867716f5.png)

Reply:
This should be working now.